server/amp/handlers/story-page.js

```diff
@@ -104,10 +104,14 @@ async function ampStoryPageHandler(
       );
     }
 
+    // Page Builder Config
+    const pbConfig = opts.getPbConfig && opts.getPbConfig instanceof Function && await opts.getPbConfig(config);
```

Comment:
As discussed earlier, rename the variable to something other than `pbConfig`. For example `publisherAmpConfig ` and `getPublisherAmpConfig `. As the framework is to be agnostic of pb.

Reply:
Sure.